source/Willow-Bootstrap-Tests/BootstrapDatepickerTest.class.st

```diff
@@ -452,7 +454,7 @@ BootstrapDatepickerTest >> testMinViewMode [
 
 	| html |
 
-	html := self renderTextFieldApplying: [ :datepicker | datepicker minViewMode: BootstrapDatepickerConstants months ].
+	html := self renderTextFieldApplying: [ :datepicker | datepicker minViewMode: datepicker constants >> #bootstrap >> #months ].
```

Comment:
Although it might be interesting to have the constants available from the datepicker component, it seems confusing that the Renoir constants require directly referencing the Namespace instead.
So, it might be better to change it here to do something like `Datepicker constants...` to make it more consistent.